File: Coding/coding modul 6/PRAK603-2410817110007-MUHAMMADHAMKAAKBARI.py
# Input bilangan n1 dan n2
n1, n2 = map(int, input().split())

# Jika n1 tidak sama dengan n2, tampilkan pesan dan berhenti
if n1 != n2:
    print("Jumlah tidak sama")
else:
    # Input angka untuk baris pertama
    baris1 = list(map(int, input().split()))
    # Input angka untuk baris kedua
    baris2 = list(map(int, input().split()))
    
    # Perkalian bilangan antara baris 1 dan baris 2
    hasil = [baris1[i] * baris2[i] for i in range(n1)]

    print("\n")
    
    # Output hasil perkalian
    print(" ".join(map(str, hasil)))

File: Coding/coding modul 6/PRAK603-2410817110007-MUHAMMADHAMKAAKBARI.c
#include <stdio.h>

int main() {
    int n1, n2;

    // Input bilangan n1 dan n2
    scanf("%d %d", &n1, &n2);

    // Jika n1 tidak sama dengan n2, tampilkan pesan
    if (n1 != n2) {
        printf("Jumlah tidak sama\n");
        return 0;
    }

    int baris1[n1], baris2[n2];

    // Input angka untuk baris pertama
    for (int i = 0; i < n1; i++) {
        scanf("%d", &baris1[i]);
    }

    // Input angka untuk baris kedua
    for (int i = 0; i < n2; i++) {
        scanf("%d", &baris2[i]);
    }

    printf("\n");

    // Perkalian bilangan antara baris 1 dan baris 2
    for (int i = 0; i < n1; i++) {
        printf("%d ", baris1[i] * baris2[i]);
    }
    printf("\n");

    return 0;
}
